sklearn/utils/_tags.py

```diff
@@ -312,6 +312,10 @@ def get_tags(estimator) -> Tags:
     -------
     tags : :class:`~.sklearn.utils.Tags`
         The estimator tags.
+
+    Notes
+    -----
+    .. versionadded:: 1.6
```

Comment:
we usually don't place these in the Notes section but rather at the end of the description, right before the Parameters section

Reply:
Done!